### PR TITLE
CVSL-1112 Moving policy config to code

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/LicencePolicyIntegrationTest.kt [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.integration
+
+import org.junit.jupiter.api.Test
+import org.springframework.http.MediaType
+import java.nio.charset.StandardCharsets.UTF_8
+
+class LicencePolicyIntegrationTest : IntegrationTestBase() {
```
Comment (andrewrlee) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/LicencePolicyIntegrationTest.kt:7]: I wrote this first and strictly asserted the json response, then moved over from the json to kotlin version so pretty confident this works as before.

File: src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicencePolicyServiceTest.kt [hunk 2]
```diff
@@ -1,29 +1,20 @@
...
-      ClassPathResource("/test_data/policy_conditions/policyV2.json").file
-    ).map { policy -> jacksonObjectMapper().readValue(policy) }
-  )
+  private val licencePolicyService = LicencePolicyService()
 
   @Test
   fun `Current policy version is returned`() {
     val policy = licencePolicyService.currentPolicy()
-    assertThat(policy.version).isEqualTo("2.0")
+    assertThat(policy.version).isEqualTo("2.1")
```
Comment (andrewrlee) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicencePolicyServiceTest.kt:13]: these tests are a bit noddy but left in to reduce the friction if we extend this area

**Review by justin-gardiner — APPROVED** [review]
lgtm

**Review by louisenorris — APPROVED** [review]
LGTM

**Pull request merged**